.eslintignore

```diff
@@ -1,2 +1,2 @@
 renderedTemplates
-
+dist/**/*.ts
```

Comment:
Is this to ignore DTS file? Was it linting the DTS files successfully before these changes?

Reply:
If linting is reporting an error, then you should fix the error, rather than disabling linting. In this case, the comment blocks are not the expected format, so they should be changed. (They are jsdoc, not tsdoc. There are minor but annoying differences.)